clouddriver-core/src/main/groovy/com/netflix/spinnaker/clouddriver/config/CloudDriverConfig.groovy

```diff
@@ -127,6 +128,7 @@ class CloudDriverConfig {
         jacksonObjectMapperBuilder.serializationInclusion(JsonInclude.Include.NON_NULL)
         jacksonObjectMapperBuilder.failOnEmptyBeans(false)
         jacksonObjectMapperBuilder.failOnUnknownProperties(false)
+        jacksonObjectMapperBuilder.modules(new Jdk8Module(), new JavaTimeModule(), new KotlinModule())
```

Comment:
Clouddriver only has one `ObjectMapper`, which is configured in core. Unfortunately, this mapper builder way of customizing things does not expose the `findAndRegisterModules` capability, for whatever reason, so we have to be explicit.